cocotb/__init__.py

```diff
@@ -53,20 +56,21 @@
 
 from ._version import __version__
 
+logging.basicConfig()
```

Comment:
Moving this outside allows me to use the cocotb logger in an interactive python session.

Reply:
This sounds like something that should probably be disabled by default for interactive sessions - perhaps `cocotb.log.setup()` or something would enable it, which would also achieve the goal of moving stuff out of `__init__.py`. That might be good as a standalone cleanup PR.